templates/corporate/development-community.md

```diff
@@ -31,52 +35,71 @@ action in the development community.
-feedback.
-
-## Getting started
-
-- Send test messages to [#test
-  here](https://chat.zulip.org/#narrow/channel/7-test-here) or as a PM
-  to yourself to avoid disturbing others.
-- Start a [new topic](/help/introduction-to-topics) unless you’re replying
-  to an existing conversation. If you see a new user talking about
-  something unrelated on an existing topic, [move the messages to an
-  appropriate new topic](/help/move-content-to-another-topic).
-- If you’re not sure where to start, introduce yourself and your
-  interests in [#new
-  members](https://chat.zulip.org/#narrow/channel/95-new-members),
-  using your name as the topic to create a dedicated space.
-- Follow the community [code of
-  conduct](https://zulip.readthedocs.io/en/latest/code-of-conduct.html).
-
-## Communication style
-
-- Converse informally; there’s no need to use titles like “Sir” or “Madam”.
-- Use [gender-neutral
-  language](https://en.wikipedia.org/wiki/Gender-neutral_language). For
-  example, avoid using a pronoun like “her” or “his” in sentences like
-  “Every developer should clean [their] keyboard at least once a
-  week.”
-
-## Getting help
-- When asking for help, provide the details needed for others to help
-  you. For example, include the **full traceback** in a [code
-  block](/help/code-blocks) (not a screenshot), a link to the code or
-  a draft PR you’re having trouble debugging, etc.
-- Ask questions on channels rather than DMing core contributors. You’ll
-  get answers faster since other people can help, and it makes it
-  possible for others to benefit from the discussion.
-- Don’t ask the same question in multiple places. Moderators read
-  every public channel, and make sure every question gets a reply.
-- Use @-mentions sparingly. Unlike IRC or Slack, Zulip’s topics make
-  it easy to see what you’re replying to, so you don’t need to mention
-  your conversation partner to make sure they will see or understand
-  your reply. Core contributors will often @-mention someone who asked
-  a question hours earlier to make sure they see the reply, but there
-  is generally no need to @-mention a core contributor unless you need
-  their timely attention.
+## How we communicate
+
+- Use **informal language**; there’s no need for titles like “Sir” or “Madam”.
+- Use **[gender-neutral
+  language](https://en.wikipedia.org/wiki/Gender-neutral_language)**. For
+  example:
+    - **Good**: Does anyone need a review on **their** PR?
+    - **Not good**: Does anyone need a review on **his** PR?
+- Aim to **communicate professionally**, using full sentences with correct spelling
+  and grammar.
+- Follow the community **[code of
+  conduct](https://zulip.readthedocs.io/en/latest/code-of-conduct.html)**.
+
+Read the [guide on how we
+communicate](https://zulip.readthedocs.io/en/latest/contributing/how-we-communicate.html)
+to learn more.
+
+## How to ask for help
+
+- Formulate a **clear question**, which includes an appropriate amount of context
```

Comment:
Would it be useful to link to https://zulip.readthedocs.io/en/stable/contributing/asking-great-questions.html here?

Reply:
I considered it and decided to just copy the parts that seemed most important here. You can see the similarity to the bullets on that page. :) https://zulip.readthedocs.io/en/stable/contributing/asking-great-questions.html#where-to-ask-your-question is also redundant with this page.

So I didn't think there was enough different content there for it to be worth it. Also slightly worried about sounding patronizing in this context, which is not just for contributors.